dll/win32/comdlg32/colordlg.c

```diff
@@ -1215,6 +1226,9 @@ static INT_PTR CALLBACK ColorDlgProc( HWND hDlg, UINT message,
 	  case WM_INITDIALOG:
 	                return CC_WMInitDialog(hDlg, wParam, lParam);
 	  case WM_NCDESTROY:
+#ifdef __REACTOS__
+	                ClipCursor(NULL); // In case closed before WM_LBUTTONUP received.
```

Comment:
What do you mean with this comment?

Reply:
KRosUser says:

> @hbelusca @katayama_hirofumi_mz : This is necessary otherwise if you hold click then press ESC or ALT+F4 to get dialog closed, the clipping remains active since ButtonUp is never called before closing....

Reply:
OK then the comment in code could be slightly reformulated as follows:
`// Ensure clipping is released, in case the dialog is closed before WM_LBUTTONUP is received.`

Reply:
```suggestion
	                // Ensure clipping is released, in case the dialog is closed before WM_LBUTTONUP is received.
	                ClipCursor(NULL);
```